Library/src/Multiplayer/MultiplayerConnection.cpp

```diff
@@ -148,22 +146,22 @@ MultiplayerConnection::~MultiplayerConnection()
 		CSP_DELETE(WebSocketClient);
 		CSP_DELETE(NetworkEventManager);
 		CSP_DELETE(ConversationSystemPtr);
+		EventBusPtr = nullptr;
```

Comment:
We need to call CSP_DELETE on the EventBus

Reply:
done